ios/gamma/split-view/RNSSplitViewHostComponentView.mm

```diff
@@ -13,6 +13,9 @@
 
 static const CGFloat epsilon = 1e-6;
 
+#define HAS_COLUMN_METRIC_CHANGED(OLD, NEW, PROPERTY_NAME, EPSILON) \
```

Comment:
I think we normally use past simple for such cases, so maybe `DID_COLUMN_METRIC_CHANGE` or even `COLUMN_METRIC_CHANGED` since it returns BOOL?

Reply:
https://github.com/software-mansion/react-native-screens/pull/3065/commits/79980f87b7ebe062fe63855e930ae916247064d1